src/app/pages/plugins/plugins-installed/plugins-installed.component.ts

```diff
@@ -103,38 +103,33 @@ export class PluginsInstalledListComponent {
       }
     },
     {
-      id: "mupgrade",
-      label: T("Upgrade"),
-      icon: "arrow_upward",
+      id: "mupupdate",
+      label: T("Update"),
+      icon: "update",
       enable: true,
       ttpos: "above",
       onClick: (selected) => {
-        const option = {
-          'plugin': true,
-        }
         const selectedJails = this.getSelectedNames(selected);
-        for (const i in selectedJails) {
-          selectedJails[i].push(option);
-        }
 
-        this.snackBar.open(T('Upgrade selected jails started.'), 'close', { duration: 5000 });
+        this.snackBar.open(T('Updating selected plugins.'), 'close', { duration: 5000 });
         this.entityList.busy =
-          this.ws.job('core.bulk', ["jail.upgrade", selectedJails]).subscribe(
+          this.ws.job('core.bulk', ["jail.update_to_latest_patch", selectedJails]).subscribe(
             (res) => {
-              if (res.result != null) {
-                if (res.result[0] && res.result[0].error != null) {
-                  this.dialogService.errorReport(T('Upgrade selected jails failed.'), res.result[0].error);
+              let message = "";
+              for (let i = 0; i < res.result.length; i++) {
+                if (res.result[i].error != null) {
+                  message = message + '<li>' + selectedJails[i] + ': ' + res.result[i].error + '</li>';
                 }
+              }
+              if (message === "") {
+                this.snackBar.open(T('Selected plugins updated.'), 'close', { duration: 5000 });
               } else {
-                for (const i in selected) {
-                  this.updateRow(selected[i]);
-                }
-                this.updateMultiAction(selected);
-                this.snackBar.open(T('Upgrade selected jails succeeded.'), 'close', { duration: 5000 });
+                message = '<ul>' + message + '</ul>';
+                this.dialogService.errorReport(T('Update of these plugins failed'), message);
```

Comment:
If this is the dialog title, can be shorter and title case:
```suggestion
                this.dialogService.errorReport(T('Plugin Update Failed'), message);
```